lib/api.js

```diff
@@ -0,0 +1,284 @@
+var BaseClient = require( "./client" ),
+	extend = require( "./extend" ),
+	util = require( "util" );
+
+
```

Comment:
If you look at the [existing code](https://github.com/scottgonzalez/node-browserstack/blob/1246998566c7d577f01b43dc56710ed1f9a198ed/lib/browserstack.js#L5), you'll see there's only one blank line here.

You don't have to worry about all the spacing issues though, I can fix those when I merge.